src/helpers/getPage.ts

```diff
@@ -1,7 +1,7 @@
 //this function allows you to pass in a node and return its pageNode
-export default function getPage(node: BaseNode): PageNode {
+export default function getPage(node: BaseNode) {
```

Comment:
```suggestion
export default function getPage(node: BaseNode): node as PageNode {
```

Reply:
Not valid..

![2020-03-26_133804](https://user-images.githubusercontent.com/39444579/77637937-798fc000-6f67-11ea-85e6-b39bcf6a603d.png)

Reply:
sorry, it should be `node is PageNode` instead of `node as PageNode`